Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Ensure go modules are enabled:
+export GO111MODULE=on
+export GOPROXY=https://proxy.golang.org
+
+# Disable CGO so that we always generate static binaries:
+export CGO_ENABLED=0
+
+# Unset GOFLAG for CI and ensure we've got nothing accidently set
+unexport GOFLAGS
+
+.PHONY: build
+build:
+	go build
+
+.PHONY: test
+test:
+	go test ./...
+
+.PHONY: coverage
+coverage:
+	go test -coverprofile=cover.out  ./...
+
+.PHONY: fmt
+fmt:
+	gofmt -s -l -w cmd pkg
+
+.PHONY: lint
+lint:
+	golangci-lint run --timeout 5m0s
+
+.PHONY: commits/check
```

Comment:
Not needed, only in ROSA for Prow

Reply:
I'd say it is probably good having the commit checks though. Can help tracking it down and helps in ensuring people are not having some difficult to interpret messages